.github/workflows/release.yml

```diff
@@ -0,0 +1,186 @@
+name: release
+
+on:
+    schedule:
+      - cron: "0 8 * * *"
```

Comment:
so all pre-releases will require user intervention, every day?

Reply:
We could create 2 environment secrets. One that requires manual approval (used for formal releases), and another one that requires no approval at all. If not, then we need to consider using some separate plugin to approve release. Maybe something like https://github.com/trstringer/manual-approval ?